ansible/inventories/test/group_vars/all/vars.yml

```diff
@@ -0,0 +1,237 @@
+  3832616438366637340a386639396561616161393033626363613964393137353665356630636330
+  38613661313031366135303033303538646561326232313466646139316366393566633734376363
+  35303164623261343532366330633730353266306631363863356631623261613336313832303433
+  363238623135326235396536643161313064
+hashicorp_ssh_password: !vault |
+  $ANSIBLE_VAULT;1.1;AES256
+  62633531346536643234303565373639626464326135653732323266396335353166346132383230
+  3333316665623861326164653162393361666665376466390a356535613332333838303037323535
+  61636665633334646564323336376363656464396466613535646666373033336336356562323032
+  6539363064373966300a303137666237613065326635643630326438646162326663363732303431
+  66333562666232376133643661336264356637636335323630313339343231306431616538363439
+  65626635323634653866636438663738626165366261363333313736386238373933343230386236
+  333266663739323034626462353738656662
+custos_ssh_password: !vault |
+  $ANSIBLE_VAULT;1.1;AES256
+  64356232636232646134616539663065393435386163373736366231373736326136663436316337
+  3538366264623564343862653330373238376161643232320a323935376361376362386432626330
+  62323464646334613231613664353963666362313133393937623430396132646364316264613061
+  3062343230636137370a353138333038636631303366653763636631313739623637363039356561
+  37643230623134303132343231613364383036353437643064636663653366306133376338623235
+  61613138373934653036643735323432313336646131336266386565376532333533383430663537
+  376666653633346237353065626463633330
+
+# pkcs12 passwords
+hashicorp_pkcs12_passphrase: !vault |
+  $ANSIBLE_VAULT;1.1;AES256
+  65646665313232613265363038633662393438376265663363333434303361656565653763313539
+  3937356538373262333031333463346338666538623566610a353232353565353566653564313239
+  37333962393564346261343837646564333737636637613861313532666431333136383836626539
+  6630353236613838340a383863333366383938353662356366333833393761383935663837353037
+  6537
+keycloak_pkcs12_passphrase: !vault |
+  $ANSIBLE_VAULT;1.1;AES256
+  63303962613033353165646639633338306538666233363735393266386432376663656536633663
+  6231316333363164383230303239366537366365346236620a396463313838373937653264343165
+  61363964376534616137653764303034616433626539306563333536316433396562353337646631
+  3536616534633033650a343730636632343764343834303933336266653063616335313962333137
+  64373562643465653532323839613237383762383637326332396330613831303961
+
+# deployment related variables
+custos_deployment_dir: "{{ user_home }}/custos-deployment"
+custos_source_dir: "{{ user_home }}/custos-source"
+custos_repo: "https://github.com/apache/airavata-custos.git"
+#custos_repo: "https://github.com/abhinav7sinha/airavata-custos.git"
+custos_git_branch: berametal
+#custos_git_branch: ansible-baremetal
+kafka_listener_port: 9092
+kafka_url: "{{ ansible_host }}:9092"
+zookeeper_client_port: 2181
+zookeeper_connection_url: localhost:2181
+zookeeper_connection_timeout: 18000
+
+custos_host: custos.scigap.org
```

Comment:
please use host file variable to substitute this

Reply:
Yes, doing it now - just to confirm - I'll add a new host - `custos.scigap.org` in the hosts.yml file - and refer the custos_host variable from there?

Reply:
something like this
```yaml
all:
  hosts:
    custos_host:
      ansible_host: custos.scigap.org
```

```
custos_host: "{{ hostvars[custos_host].ansible_host }}"
```

Reply:
yes

Reply:
Understood thanks!

Reply:
Done!